profile_library/hampton bay/HB-10521-HS/model.json

```diff
@@ -0,0 +1,16 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-01-30T12:14:50.534799",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly PM Mini Gen3",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.3:docker"
+  },
+  "name": "HB-10521-HS",
```

Comment:
Please put the marketed name here. That will be shown in `name` column in https://library.powercalc.nl.
Check the other profiles for examples

Reply:
Here is what is shown in the Hubspace app.

![2025-01-30 12 18 58](https://github.com/user-attachments/assets/74317524-5009-4142-a6bb-75c45301f6fb)

Reply:
@challgren current name which you have put now in model.json is perfect.